.github/workflows/gnovm.yml

```diff
@@ -1,22 +1,27 @@
-name: gnovm
+name: GnoVM CI Suite
```

Comment:
Also redundant

Reply:
These names matter because they show up as the names of the workflows; it's better if they're compact

Reply:
Resolved as part of:

[d0fe708](https://github.com/gnolang/gno/pull/3316/commits/d0fe7083e0923616594c45ec79486ddbf74c325c)